src/components/BRadio/BRadio.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <div class="container" :class="{ disabled: disabled }">
+    <div role="radiogroup" :aria-labelledby="groupId">
+      <label :id="groupId" class="description">
+      {{ groupLabel }}<span v-if="required" aria-hidden="true">*</span>
+      </label>
+      <div class="button-group" role="radiogroup">
+        <div v-for="(radio, index) in radios" class="button" :key="index">
```

Comment:
ok, I agree the key should be on the same element that the v-for is in, I guess my main question should have been, why is the v-for out here on the div and not on the input directly?  You want all those seperate div's?

Reply:
nvm there is no closing input tag..

Reply:
wait!  I know!  you can loop over label tags and put the the input inside of the label?

Reply:
something like this `<label><input type='radio'> Label</lable>` and put the v-for on the label itself, it would eliminate all the divs

Reply:
I do want them to be separate divs.  The top div has a flex-direction of column, and each button has a flex-direction of row.

Reply:
The input is self closing.

Reply:
This was a change made to have the buttons displayed vertically and have the entire group be properly responsive.

Reply:
gotcha